idseq_dag/steps/run_lzw.py

```diff
@@ -1,10 +1,29 @@
 from idseq_dag.engine.pipeline_step import PipelineStep
 import idseq_dag.util.command as command
+from command import run_in_subprocess
 import idseq_dag.util.log as log
 import idseq_dag.util.count as count
+import idseq_dag.util.fasta as fasta
+from idseq_dag.util.thread_with_result import ThreadWithResult, execute_all
+from multiprocessing import cpu_count
+from typing import Iterator
+
 
 class PipelineStepRunLZW(PipelineStep):
 
+    MAX_SUBPROCS = 16
```

Comment:
why 16?

Reply:
That's a good question.   The limit here may depend on other bottlenecks besides CPU, and I don't know yet what those are.   The value 16 is our current idea of infinity on the 32 vCPU instances where this runs.   It's also a good sane value in case there is a docker/cuser mis-config where the computation is in fact constrained to only a fraction of the available CPUs on the instance, in which case this would become a gross anti-optimization, proportional to the degree of parallelism, so it's good to try to keep it reasonable.  We can obviously revise this overtime as we gather information.